package.json

```diff
@@ -80,11 +80,15 @@
     "clean": "git clean -fdX",
     "prepublishOnly": "node ./tools/build.js",
     "watch": "babel ./src -d . --ignore __tests__ --watch",
-    "lint": "eslint src/",
+    "lint": "eslint src/ ",
+    "lint:fix": "eslint src/ --fix",
```

Comment:
I told to try `npm run lint -- --fix` and that didn't work.. but lets also try `npm run lint --fix` if that works we can remove this.

Reply:
why do we want to remove this?

Reply:
if the same can be achieved by directly passing a param to the existing one, isn't it redundant.. please let me know if there are any specific reasons for having it

Reply:
for convenience. try to run npm run lint -- --fix everytime will be bit hard, and even for the other contributors it will be easier if we have a specific script as such..

Reply:
lets merge this pr first, anything you want me to change other than the test cases, i ll do as part of the other open PR